protos/xyz/block/ftl/v1/ftl.proto

```diff
@@ -229,3 +229,23 @@ service RunnerService {
   // Initiate a deployment on this Runner.
   rpc DeployToRunner(DeployToRunnerRequest) returns (DeployToRunnerResponse);
 }
+
+enum MetricType {
+  COUNTER = 0;
+  GAUGE = 1;
+  HISTOGRAM = 2;
+}
+
+message SendMetricRequest {
+  string name = 1;
+  MetricType type = 2;
+
+  // All of the datapoints for a give metric name, which could include multiple verb/module combinations.
+  bytes datapoints = 3;
```

Comment:
We should discuss this. The `proto` definition is lighter weight than the `db` definition. The thought here is that this would be fast to marshal with multiple datapoints included in each `SendMetricRequest`, Then the observability service can just look at the type and pull the `attributes`, `start/end time`, and `value` out of the `datapoints` array